CMakeLists.txt

```diff
@@ -70,6 +72,8 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR}
                     ${X11_INCLUDE_DIR}
                     ${include_directory})
 
+include_directories(SYSTEM ${LIBUSB_1_INCLUDE_DIRS})
```

Comment:
Removes libusb warnings from compile. This is part of a separate pull request.

Reply:
you probs want to remove this change from here then and just let it get picked up by the other one

Reply:
Ok